### DISCUSSION TITLE
while True: learn() (機器學習、電玩遊戲)

### DISCUSSION THREAD

**Discussion by 109302013evantsao** [1101-GS4719-C/Python-Community-GS4719-C#223] [General]
while True: learn()是一個在多平台上的程式和機器學習相關遊戲，當初免費的時候領來玩了一下，發現還挺好玩的，我認為這遊戲有這於邏輯思考，把機器學習變得更為可視化、圖形化，既有趣又能同時學習，在此推薦給大家(BTW現在特價中喔) [連結])(https://store.steampowered.com/app/619150/while_True_learn/?l=tchinese)

**Comment by yt7l0** [comment]
所以主要是訓練邏輯思考嗎，還是要繼續寫程式QQ
